packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -90,9 +91,15 @@ export default class ContentSwitcher extends React.Component {
             selectedIndex: nextIndex,
           },
           () => {
-            const switchRef = this._switchRefs[nextIndex];
+            const child = children[this.state.selectedIndex];
+            const switchRef = this._switchRefs[this.state.selectedIndex];
             switchRef && switchRef.focus();
-            this.props.onChange(data);
+            this.props.onChange({
+              ...data,
```

Comment:
This introduces the unwanted attribute `key` (see line 80 above). Since the attributes will be overwritten completely by line 99-101, this line is not necessary.